console/src/app/components/features/features.component.ts

```diff
@@ -84,134 +101,58 @@ export class FeaturesComponent {
   public validateAndSave() {
     this.featureService.resetInstanceFeatures().then(() => {
```

Comment:
What is your opinion on using the Buf featureService instead of the old one?

Reply:
Also I'm not sure if resetting the features and then setting them makes sense. Wouldn't this always reset flags which are not in the console?

Reply:
true we can remove this.. we had this to reset them to inherit again but as I have removed that toggle entirely we do not need it anymore